src/app/search/items/search-filters/simple.ts

```diff
@@ -95,6 +95,24 @@ const simpleFilters: ItemFilterDefinition[] = [
     description: tl('Filter.VendorItem'),
     filter: () => (item) => Boolean(item.vendor),
   },
+  {
+    keywords: 'owned',
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    description: 'Filter.Owned' as any,
```

Comment:
You shouldn't disable lints. In this case the correct version would be to follow the example of all the other filters:
```suggestion
    description: tl('Filter.Owned'),
```